api/client.yaml

```diff
@@ -2022,13 +2059,90 @@ components:
           example: '2016-08-29T09:12:33.001Z'
       required:
         - customerID
-        - firstName
-        - lastName
         - type
         - status
         - email
         - createdAt
         - lastModified
+      discriminator:
+        propertyName: type
+        mapping:
+          individual: '#/components/schemas/CustomerIndividual'
+          business: '#/components/schemas/CustomerBusiness'
+    CustomerIndividual:
+      allOf:
+        - $ref: '#/components/schemas/Customer'
+        - type: object
+          properties:
+            createdAt:
+              type: string
+              format: date-time
+              example: '2016-08-29T09:12:33.001Z'
+            lastModified:
+              type: string
+              format: date-time
+              description: Last time the object was modified
+              example: '2016-08-29T09:12:33.001Z'
```

Comment:
Hmmm it looks like these fields are already on the `Customer` object:
https://github.com/moov-io/customers/blob/d3b7c444bb9382d950ffb72d9ea61852af5e1186/api/client.yaml#L2051

Reply:
Yes, you're right.  However, if I leave them off these other schemas, for some reason the generator does not import "time" in these models.  So the workaround was to include these in each of the schemas and that works

Reply:
Ah I see.

Add `goimports` to the makefile to import `time`:
```bash
.PHONY: client
client:
	@rm -rf ./pkg/client
	docker run --rm \
		-u $(USERID):$(GROUPID) \
		-v ${PWD}:/local openapitools/openapi-generator-cli:v4.3.1 batch -- /local/.openapi-generator/client-generator-config.yml
	rm -f ./pkg/client/go.mod ./pkg/client/go.sum
	goimports -w ./pkg/client/
	gofmt -w ./pkg/client/
	go build github.com/moov-io/customers/pkg/client
```

Reply:
Ah, interesting.  I'll try adding `goimports` as you said!